assets/svgs.js

```diff
@@ -0,0 +1,7 @@
+export const EmailIcon = (props) => {
```

Comment:
Can you remove this, please? It is currently not related to the storybook story.